### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by anaymehta**
updated main.py to include blackbox

### CODE REVIEW

**Comment by anaymehta** [comment]
Waiting on a review before merging @JacobSandler @RupinMittal

File: daq/panda/main.py
Comment (KensonN) [inline daq/panda/main.py]: If you set the path as "os.path.dirname(\_\_file__)"  it will use the current directory as the save path so we won't have to change it when we move the code to the RasPi.
```suggestion
  savePath = os.path.dirname(__file__)
```

**Review by KensonN — APPROVED** [review]

**Pull request merged**
